packages/dds/tree/src/simple-tree/api/tree.ts

```diff
@@ -242,6 +242,7 @@ export interface TreeBranchAlpha extends TreeBranch {
 	 * - Constraints will apply to the outermost transaction. Constraints are applied per commit and there will be one commit generated
 	 * for the outermost transaction which includes all inner transactions.
 	 * - Undo will undo the outermost transaction and all inner transactions.
+	 * - If a label is provided in the params, it will be used for the outermost transaction.
```

Comment:
This makes it sound like providing a label to an _inner_ transaction might possible override an existing label for the _outer_ transaction. What we want to say is something like "only the label for the outermost transaction does anything, we ignore all labels for inner/nested transactions".